metrics/lastseen/redis.go

```diff
@@ -0,0 +1,39 @@
+package lastseen
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-redis/redis"
```

Comment:
we should use the same lib as in the other code base : https://github.com/wrapp/merchant-api/blob/master/clients/redis.go#L6

Reply:
That one has been archived !
and moved to https://github.com/gomodule/redigo instead.
So maybe we should use that one instead in merchant-api.
This is what is starred and linked on the https://redis.io/clients#go

Reply:
When I implemented it in `merchants-stats` go-redis lacked some features like pipelining and was not great in terms of perf (I did the first implementation in go-redis but it was very poor). Nice to see it has more feature now.

I personally don't care that much if we choose go-redis from now on : I just wanted to point out we were using redigo until now and it's a bad idea IMHO to have too many dependency that covers the same domain :)